src/modules/auth/login.controller.ts

```diff
@@ -132,4 +132,28 @@ export class LoginController {
 
     return res.send({ token, refreshToken });
   }
+
+  @Get('auth/status')
+  async status(@Req() req: Request) {
```

Comment:
This check is supposed to ensure that access token is matching to requester DID instead of to DID from previous session. I guess it should just return user extracted from `@User` decorator

Reply:
I will update it